### PR TITLE
update doc on modifying the ISO installer

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

File: docs/develprocess.md [hunk 1]
```diff
@@ -650,12 +650,17 @@ bunzip2 < ../install.img | cpio -idm
...
 ```
 
-### Navigate in the installer's filesystem
+#### Navigate in the installer's filesystem
 
 If you want to use commands in the installer's filesystem context, as root:
 ```
 chroot install/
 ```
+You'll also need to mount `urandom` in your chrooted dir:
```
Comment (stormi) [inline docs/develprocess.md:659]: Let's explain that the commands below have to be run from outside the chroot and that it's needed for using `rpm` and `yum` commands?

File: docs/develprocess.md [hunk 2]
```diff
@@ -665,15 +670,18 @@ rpm -qa | sort
 
 Exit chroot with `exit` or Ctrl + D.
 
-### Alter the filesystem
+#### Alter the filesystem
 
 Using chroot as explained above, you can easily remove, add or update RPMs in the installer's filesystem.
 
+> This modifies the installer filesystem, not the host!
```
Comment (stormi) [inline docs/develprocess.md:677]: You can use a `:::warning` or `:::tip` section here

File: docs/develprocess.md [hunk 3]
```diff
@@ -702,12 +710,11 @@ Read [the usual warnings about the installation of third party RPMs on XCP-ng.](
 
 To achieve this:
 * Change the RPMs in the `Packages/` directory. If you add new packages, be careful about dependencies, else they'll fail to install and the whole installation process will fail.
-* Copy the file called `repodata/{varying_checksum_here}-groups.xml to a temporary location such as `/tmp/groups.xml`
 * Modify it to add or remove RPMs from the groups. There are two groups and both will be installed, so it's not very important if you don't know which one to modify. Just pick one. You don't need to add all the dependencies: they will be pulled automatically if you made them available in `Packages/`.
```
Comment (stormi) [inline docs/develprocess.md:713]: This isn't true anymore.

Let's replace it with "If you need to add new RPMs not just replace existing ones, they need to be pulled by another existing RPM as dependencies. If there's none suitable, you can add the dependency to the `xcp-ng-deps` RPM (link to the git repo)"

Comment (benjamreis) [reply docs/develprocess.md:713]: Where?

**Pull request merged**